### DISCUSSION TITLE
Issue user status recover

### DISCUSSION THREAD

**Discussion by bromlecornu** [authelia/authelia#7442] [Q&A]
Hello, I've installed authelia for my frigate access.

I have an issue when I want to access to it. When I'm on my lan I have this message. See pic
![Screenshot_20240619_224743_Chrome](https://github.com/authelia/authelia/assets/33418528/56fc925b-f532-4c1a-aff8-a28dcec66af5)

And when I'm out, it doesn't, server too many time to response. 

Any idea please

Thx

**Comment by glime-ai** [comment]
Please note this response was created using generative AI leveraging previously available information from the repository. This information may be helpful however it is our strong recommendation that you carefully consider the information before blindly applying it.

---

It seems like you are experiencing issues with accessing Authelia both from within your LAN and externally. Here are a few steps you can take to troubleshoot and potentially resolve the issue:

1. **Check Network Configuration:**
   - Ensure that your network configuration allows access to Authelia from both internal and external networks. Verify that the necessary ports are open and properly forwarded if you are accessing it from outside your LAN.

2. **DNS and Hosts Configuration:**
   - Make sure that your DNS settings are correctly configured. If you are using a local DNS resolver like AdGuard Home, ensure that it points to the correct IP addresses for your services.
   - Verify that your `/etc/hosts` file (or equivalent) has the correct entries for your domain names.

3. **Authelia Configuration:**
   - Review your `configuration.yml` file to ensure that the server address and other settings are correctly configured. For example:
     ```yaml
     server:
       address: 'tcp://:9091'
     ```
   - Ensure that the `log` settings are correctly configured to help you debug any issues:
     ```yaml
     log:
       level: 'debug'
       format: 'text'
       file_path: /config/authelia.log
     ```

4. **Proxy Configuration:**
   - If you are using a reverse proxy like Nginx or Traefik, ensure that the proxy configuration is correct and that it properly forwards requests to Authelia. Check for any misconfigurations that might be causing the issue.

5. **Check Logs:**
   - Check the Authelia logs for any error messages or warnings that might give you more insight into what is going wrong. The logs can be found at the path specified in your configuration file (e.g., `/config/authelia.log`).

6. **Performance Issues:**
   - If you are experiencing performance issues, such as the server taking too long to respond, consider checking the resource usage on your server. Ensure that your server has enough CPU and memory to handle the load.

7. **External Access:**
   - For external access issues, ensure that your firewall and router settings allow traffic to reach your Authelia instance. Verify that your public IP address and port forwarding settings are correct.

If you continue to experience issues, please provide more details about your setup, including your Authelia configuration, proxy configuration, and any relevant log entries. This will help in diagnosing the problem more accurately.

This comment was generated by [Glime](https://glimelab.ai/?utm_source=6fsHkcyufjQLaB&utm_medium=github).

**Comment by james-d-elliott** [comment]
What do the authelia logs show?

**Reply by bromlecornu** [reply]
Hi,
there is my log file

`time="2024-06-20T06:26:07Z" level=info msg="Storage schema is being checked for updates"
time="2024-06-20T06:26:07Z" level=info msg="Storage schema is already up to date"
time="2024-06-20T06:26:07Z" level=debug msg="Create Server Service (metrics) skipped"
time="2024-06-20T06:26:07Z" level=info msg="Startup complete"
time="2024-06-20T06:26:07Z" level=info msg="Listening for non-TLS connections on '[::]:9091' path '/'" server=main service=server
time="2024-06-20T06:40:37Z" level=error msg="Error occurred retrieving user session" error="unable to retrieve session cookie domain provider: no configured session cookie domain matches the url 'http://(ipdockervm):9091/api/state'" method=GET path=/api/state remote_ip=192.168.20.211 stack="github.com/authelia/authelia/v4/internal/handlers/handler_state.go:16  StateGET\ngithub.com/authelia/authelia/v4/internal/middlewares/bridge.go:54      handleRouter.(*BridgeBuilder).Build.func5.1\ngithub.com/authelia/authelia/v4/internal/middlewares/headers.go:65     SecurityHeadersCSPNone.func1\ngithub.com/authelia/authelia/v4/internal/middlewares/headers.go:91     SecurityHeadersNoStore.func1\ngithub.com/authelia/authelia/v4/internal/middlewares/headers.go:30     SecurityHeadersBase.func1\ngithub.com/fasthttp/router@v1.5.1/router.go:441                        (*Router).Handler\ngithub.com/authelia/authelia/v4/internal/middlewares/log_request.go:14 handleRouter.LogRequest.func40\ngithub.com/authelia/authelia/v4/internal/middlewares/errors.go:38      RecoverPanic.func1\ngithub.com/valyala/fasthttp@v1.54.0/server.go:2379                     (*Server).serveConn\ngithub.com/valyala/fasthttp@v1.54.0/workerpool.go:224                  (*workerPool).workerFunc\ngithub.com/valyala/fasthttp@v1.54.0/workerpool.go:196                  (*workerPool).getCh.func1\nruntime/asm_amd64.s:1695                                               goexit"
`

Just this.

**Reply by james-d-elliott** [reply]
You will either need to access it via a FQDN which is configured in your session section, or configure the headers required for operation: https://www.authelia.com/integration/proxies/introduction/#required-headers

**Reply by bromlecornu** [reply]
Sorry I don't understand how to do

This my configuration.yml

`## The theme to display: light, dark, grey, auto.
theme: dark

server:
  port: 9091

log:
  ## Level of verbosity for logs: info, debug, trace.
  level: debug
  file_path: /config/authelia.log

## The secret used to generate JWT tokens when validating user identity by email confirmation. JWT Secret can also be
## set using a secret: https://www.authelia.com/c/secrets
jwt_secret: my_very_important_secret

authentication_backend:
  file:
    path: /config/users.yml
    watch: false
    search:
      email: false
      case_insensitive: false
    password:
      algorithm: argon2
      argon2:
        variant: argon2id
        iterations: 3
        memory: 65536
        parallelism: 4
        key_length: 32
        salt_length: 16

totp:
  ## Disable TOTP.
  disable: true

session:
  ## The name of the session cookie.
  name: authelia_session

  ## The domain to protect.
  ## Note: the authenticator must also be in that domain.
  ## If empty, the cookie is restricted to the subdomain of the issuer.
  domain: xxxxxx.xx
  same_site: lax
  secret: unsecure_session_secret
  expiration: 1h
  inactivity: 5m
  remember_me_duration:  1M

storage:
  encryption_key: my_very_important_secret
  local:
    path: /config/db.sqlite3

access_control:
  default_policy: one_factor
  rules:
    - domain: authelia.xxxxx.xx
      policy: one_factor
    resources:
     - - header: X-Forwarded-Host: authelia.xxxxx.xx

notifier:
  filesystem:
    filename: /var/lib/authelia/emails.txt
`

Thx for your help

**Reply by james-d-elliott** [reply]
Are you using a proxy?

**Comment by electricmessiah** [comment]
I just ran into this issue. I did nothing but shut down the docker container for a day, then on reboot this error occurs. I'm not behind a proxy and I am accessing it directly via 192.168.1.10:9091. 

`time="2024-08-17T11:15:06-04:00" level=error msg="Error occurred retrieving user session" error="unable to retrieve session cookie domain provider: no configured session cookie domain matches the url 'http://192.168.1.10:9091/api/state'"`

I've tried multiple browsers, so it's not a cookie thing that I can tell. Surely there's a way to clear whatever the issue is and resume.

**Reply by james-d-elliott** [reply]
Do you have a domain configured for `192.168.1.10`?
